src/component/graph/Graph.ts

```diff
@@ -8,35 +8,24 @@ import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
 export default class Graph {
   private mxClient: any = MxGraphFactoryService.getMxGraphProperty('mxClient');
   private mxUtils: any = MxGraphFactoryService.getMxGraphProperty('mxUtils');
-  private mxGraph: any = MxGraphFactoryService.getMxGraphProperty('mxGraph');
-  private mxGraphModel: any = MxGraphFactoryService.getMxGraphProperty('mxGraphModel');
 
-  protected readonly graph: mxgraph.mxGraph;
+  public readonly graph: mxgraph.mxGraph;
```

Comment:
public only for test, can do something else?

Reply:
as decided on call for now it stays this way